qiskit/transpiler/runningpassmanager.py

```diff
@@ -144,17 +145,26 @@ def _do_pass(self, pass_, dag, options):
         Raises:
             TranspilerError: If the pass is not a proper pass instance.
         """
+        if isinstance(pass_, BasePass):
+            # First, do the requires of pass_
+            for required_pass in pass_.requires:
+                dag = self._do_pass(required_pass, dag, options)
 
-        # First, do the requires of pass_
-        for required_pass in pass_.requires:
-            dag = self._do_pass(required_pass, dag, options)
+            # Run the pass itself, if not already run
+            if pass_ not in self.valid_passes:
+                dag = self._run_this_pass(pass_, dag)
 
-        # Run the pass itself, if not already run
-        if pass_ not in self.valid_passes:
-            dag = self._run_this_pass(pass_, dag)
+                # update the valid_passes property
+                self._update_valid_passes(pass_)
 
-            # update the valid_passes property
-            self._update_valid_passes(pass_)
+        # if provided a nested flow controller
+        elif isinstance(pass_, FlowController):
+
+            if not isinstance(pass_.condition, partial):
+                pass_.condition = partial(pass_.condition, self.fenced_property_set)
+
+            for _pass in pass_:
+                self._do_pass(_pass, dag, pass_.options)
 
```

Comment:
```suggestion
        else:
                raise(f"Expecting type BasePass or FlowController, got {type(pass_)}.")
```

Reply:
Usually, people add a "Raises:" section to the docstring for this. But, I find it overly verbose. In fact the docstring guidelines that we follow do *not* require such a section in this situation. But, the linting tools (maybe pylint) may nonetheless complain that the section is missing.